notebooks/pandas.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
Would you mind moving this notebook to a new subdirectory under notebooks (i.e. `notebooks/analysis/`)?